packages/Molecules/Form/fields/fieldRplclearform.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <button class="rpl-clearform" @click="clearForm">
+    <rpl-text-icon :text="schema.buttonText" symbol="cross_circle" placement="before" color="danger" size="m" />
```

Comment:
An alternative to setting size to 'm' would be to set size="0.8333". By doing this we can remove max-width and max-height values from the SCSS below. I'm not too fussed either way, but this is how it has been done in other cases.